src/main/steps/sibling/placement-order/get.test.ts

```diff
@@ -0,0 +1,206 @@
+import { mockRequest } from '../../../../test/unit/utils/mockRequest';
+import { mockResponse } from '../../../../test/unit/utils/mockResponse';
+import { generateContent } from '../placement-order-number/content';
+
+import SiblingPlacementOrderGetController from './get';
+
+describe('SiblingPlacementOrderGetController', () => {
+  let controller;
+  let req;
+  let res;
+
+  beforeEach(() => {
+    Date.now = jest.fn(() => +new Date('2021-01-01'));
+    controller = new SiblingPlacementOrderGetController(__dirname + '../../common/template', generateContent);
+    req = mockRequest({ session: { userCase: { siblings: [] } } });
+    res = mockResponse();
+  });
+
+  describe('when there is no selectedSiblingPoId in userCase', () => {
+    test('should generate random placementOrderId', async () => {
+      req.locals.api.triggerEvent.mockResolvedValue({
+        selectedSiblingPoId: '1609459200000',
+        placementOrders: [{ placementOrderId: '1609459200000' }],
+      });
+      await controller.get(req, res);
+      expect(req.session.userCase.selectedSiblingPoId).toBe('1609459200000');
+    });
+  });
+
+  describe('when there is a selectedSiblingPoId in userCase', () => {
+    test('should not generate random placementOrderId', async () => {
+      req = mockRequest({ session: { userCase: { selectedSiblingPoId: 'MOCK_PLACEMENT_ORDER_ID' } } });
+      req.locals.api.triggerEvent.mockResolvedValue({
+        selectedSiblingPoId: 'MOCK_PLACEMENT_ORDER_ID',
+        placementOrders: [{ placementOrderId: 'MOCK_PLACEMENT_ORDER_ID' }],
+      });
+      await controller.get(req, res);
+      expect(req.session.userCase.selectedSiblingPoId).toBe('MOCK_PLACEMENT_ORDER_ID');
+    });
+  });
+
+  describe('when there is no placementOrder with selectedSiblingPoId in userCase', () => {
+    test('should create a blank placementOrder with generated placementOrderId', async () => {
+      req.locals.api.triggerEvent.mockResolvedValue({
+        selectedSiblingPoId: '1609459200000',
+        placementOrders: [{ placementOrderId: '1609459200000' }],
+      });
+      await controller.get(req, res);
+      expect(req.session.userCase.placementOrders).toEqual([{ placementOrderId: '1609459200000' }]);
```

Comment:
this file should be specific to sibling placement orders
please update `req.session.userCase.placementOrders` it's not correct